components/gc-features/samples/features.html

```diff
@@ -0,0 +1,31 @@
+{%- comment -%}
+
+++++++++++++++++++++++++++++++++++
++ List of features
+++++++++++++++++++++++++++++++++++
+
+{%- endcomment -%}
+
+<section class="gc-features">
```

Comment:
Side note: the gc-features is also about the use of this class "gc-prtts" as showed in the news-en.html here: https://github.com/wet-boew/themes-dist/blob/v4.0.28-gcweb/news-en.html#L205-L208